frontend/src/__tests__/cypress/cypress/tests/mocked/modelRegistry/modelRegistry.cy.ts

```diff
@@ -178,7 +178,10 @@ describe('Model Registry core', () => {
         .contains(
           'A machine learning model trained to detect fraudulent transactions in financial data',
         );
-      registeredModelRow.findOwner().contains('Author 1');
+      registeredModelRow.findOwner().should(($owner) => {
+        const text = $owner.text();
+        expect(text).to.be.oneOf(['Author 1', '-']);
```

Comment:
I think we should test both ways here, and that we'll want to test both ways (that the author shows up if the author prop is populated, that the '-' shows up if not).

Reply:
That line of code started failing with the new change, and I wasn't quite able to figure out why ( findOwner has the same textId 'registered-model-owner' so it should be good) . It's complaining that :"Timed out retrying after 10000ms: Expected to find content: 'Author 1' within the element: <p> but never did". Do you have maybe a suggestion what can be missing?

Reply:
It looks like you haven't updated the mock types and data. They still have the author in the modelVersion.

Reply:
So if I understood correctly I should delete "mockModelVersion({ author: 'Author 1' })" line 70 in that test file? - I tried, that didn't help. I also tried to delete that field from all the 3 parts within MockModelVersionType but then the test suite didn't even run.

Reply:
You'll need to also add the author prop to the ModelRegistry where your code is looking for it.

As for the tests not running in MockModelVersionType after removing all the model versions. I'm not sure, is that completely removed from everywhere or just within this case? If removed generally and tests fail after mocking that way, then there must be some test that is still falsely depending on that.

Reply:
looks like you need both `owner` and `author` on the `mockRegisteredModel` stuff

Reply:
ah, actually i can't do that as the `mockRegisteredModel.ts` file wasn't touched yet. but basically:

<img width="381" alt="image" src="https://github.com/user-attachments/assets/58729a8f-3a75-4aae-ba7f-d316b5fc8b0f">


then you could update the calls to `mockRegisteredModel` in `modelRegistry.cy.ts` to whatever owner/author you want, and the test to match. could use `Author 1` if you want, but i think something with "Owner" would make more sense for the owner field so people don't get the two mixed up in reviewing/making tests.

Reply:
or is there even an author field? you may have just had a typo below in`ModelDetailView`

Reply:
ah, actually i can't do that as the `mockRegisteredModel.ts` file wasn't touched yet. but basically:

<img width="381" alt="image" src="https://github.com/user-attachments/assets/58729a8f-3a75-4aae-ba7f-d316b5fc8b0f">


then you could update the calls to `mockRegisteredModel` in `modelRegistry.cy.ts` to whatever owner/author you want, and the test to match. could use `Author 1` if you want, but i think something with "Owner" would make more sense for the owner field so people don't get the two mixed up in reviewing/making tests.

or is there even an author field? you may have just had a typo below in`ModelDetailView`

Reply:
Sorry, I just wrapped my head around the story and changes a bit more. I think you need to still treat `author` the same way and keep it in modelVersion and just add `owner` to the registeredModel. So the RM has an "owner" while the version has an "author".

Maybe @mturley has some input on this

Reply:
Yes, it's confusing but the RegisteredModel has an `owner` but no author and the ModelVersion (and ModelArtifact, though that's not relevant) has an `author` but no owner. Currently in the UI we are showing "Owner" and under the hood we're using the version's `author` field.

There are two changes that prompted this story:
1. The database now has `owner` on RegisteredModel, which previously didn't have either owner or author (which is why we were fetching the author from the model's latest version).
2. The UX microcopy is being updated so that we call it in the UI what it is in the database -- for models we call it "Owner" and use the `.owner` field, and for versions we'll call it "Author" and use the `.author` field.

Reply:
That latter change to use "Author" in the UI wasn't originally part of this Jira issue, it was from a followup discussion, but I think making that change in this PR would be least confusing. Is that ok @YuliaKrimerman ?

Reply:
Actually --- scratch that last point, sorry. We don't have to touch anything with how Versions display the owner/author, that's out of scope here.